app-text/wiki/wiki-1.4.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GOLANG_PKG_IMPORTPATH="github.com/walle"
+
```

Comment:
Package `wiki` has support for unit testing too, you can add `GOLANG_PKG_HAVE_TEST=1`.
Please, make sure that the ebuild is compiling correctly when executed with `FEATURES="test"`; sometimes unit tests pull in extra dependencies.